README.md

```diff
@@ -58,7 +58,9 @@ In order for Thimble to be installed correctly, the following dependencies need
 - [Brackets (Bramble)](#installing-brackets-bramble)
 - Virtualbox (version 5.1 or later) [[download](https://www.virtualbox.org/wiki/Downloads)]
 - Vagrant (version 1.9 or later) [[download](https://www.vagrantup.com/downloads.html)]
-  - __Note:__ On Windows machines, you may need to restart your computer after installing Vagrant for it to be fully usable.
+  - __Note:__ On Windows machines, you may need to restart your computer after installing Vagrant for it to be fully usable.  
+  Avoid installation in directories containing *spaces*.  Vagrant is written in Ruby which has issues with directory names containing spaces, so be sure that your VAGRANT_HOME environment variable does not contain any spaces (i.e. a user home folder w/ spaces).  You can set VAGRANT_HOME via Control Panel > Advanced > Environment variables or set it from command prompt with [setx](https://technet.microsoft.com/en-us/library/cc755104(v=ws.11).aspx) command.  
```

Comment:
We want to be clear here, so we'll want to say `Control Panel > System > Advanced system settings > Environment Variables > System Variables`.